apps/website/src/composables/wallet.ts

```diff
@@ -19,17 +19,17 @@ const ethersProviderList = ['MetaMask', 'CoinbaseWallet']
 // Test iotex send to address: acc://06da5e904240736b1e21ca6dbbd5f619860803af04ff3d54/acme
 
 export default function useWallet() {
-  const { getIoPayAccounts, sendIoPayTransaction } = useIopay(toAddress, amount)
+  const { getIoPayAccounts, sendIoPayTransaction } = useIoPay(toAddress, amount)
   const ethereum: any = window.ethereum
   const availableProviders = ref<BrowserProviders>(
     getBrowserProviders(ethereum)
   )
-  type ProviderString = keyof BrowserProviders | 'IoPay' | ''
+  type ProviderString = keyof BrowserProviders | 'IoPay' | '' // Why are we adding this type here and not in global namespace?
```

Comment:
Could move to `@/types/ProviderString.ts`

You could see it evolve to something like `keyof BrowserProviders | keyof NativeProviders | ''`

Reply:
ok will do this.

Reply:
@shanejearley Semi-resolved. Check Slack.